.meta/_partials/_kafka_common.toml

```diff
@@ -0,0 +1,43 @@
+[<%= namespace %>.options.bootstrap_servers]
+type = "string"
+common = true
+examples = ["10.14.22.123:9092,10.14.23.332:9092"]
+required = true
+description = """\
+A comma-separated list of host and port pairs that are the addresses of the \
+Kafka brokers in a \"bootstrap\" Kafka cluster that a Kafka client connects \
+to initially to bootstrap itself.\
+"""
+
+[<%= namespace %>.options.compression]
+type = "string"
+common = true
+examples = ["none","gzip"]
```

Comment:
You don't need this if you provide an `enum`.

Reply:
Fixed